### ISSUE TITLE
Escape table names

### ISSUE THREAD

**Issue by jens-ox** [drizzle-team/drizzle-kit-mirror#19]
When generating migrations, table names don't get escaped. For example, if a table is to be called "user" and has some foreign key constraint, this will be generated:

```sql
DO $$ BEGIN
 ALTER TABLE user ADD CONSTRAINT user_office_id_office_id_fk FOREIGN KEY ("office_id") REFERENCES office("id") ON DELETE no action ON UPDATE no action;
EXCEPTION
 WHEN duplicate_object THEN null;
END $$;
```

`user` is a reserved word, but a valid table name. Escaping it makes this a valid query:

```sql
DO $$ BEGIN
 ALTER TABLE "user" ADD CONSTRAINT user_office_id_office_id_fk FOREIGN KEY ("office_id") REFERENCES office("id") ON DELETE no action ON UPDATE no action;
EXCEPTION
 WHEN duplicate_object THEN null;
END $$;
```

**Comment by AndriiSherman** [comment]
Thanks! Was fixed [here](https://github.com/drizzle-team/drizzle-kit-mirror/issues/16)
Just isn’t in latest yet
Will check all statements for quotes in next releases

**Comment by AndriiSherman** [comment]
@jens-ox fixed in `drizzle-kit@0.17.4` 🎉
